pkg/bff/models/v1/testdata/default_registration_form.json

```diff
@@ -1,116 +1,116 @@
+        "post_code": "",
+        "town_name": "",
+        "town_location_name": "",
+        "district_name": "",
+        "country_sub_division": "",
+        "address_line": [],
+        "country": ""
       }
+    ],
+    "customer_number": "",
+    "national_identification": {
+      "national_identifier": "",
+      "national_identifier_type": "NATIONAL_IDENTIFIER_TYPE_CODE_LEIX",
+      "country_of_issue": "",
+      "registration_authority": "RA777777"
     },
-    "trixo": {
-      "primary_national_jurisdiction": "",
-      "primary_regulator": "",
-      "financial_transfers_permitted": "no",
-      "other_jurisdictions": [],
-      "has_required_regulatory_program": "no",
-      "conducts_customer_kyc": false,
-      "kyc_threshold": 10,
-      "kyc_threshold_currency": "USD",
-      "must_comply_travel_rule": false,
-      "applicable_regulations": [
-        "FATF Recommendation 16"
-      ],
-      "compliance_threshold": 3000,
-      "compliance_threshold_currency": "USD",
-      "must_safeguard_pii": false,
-      "safeguards_pii": false
+    "country_of_registration": ""
+  },
+  "contacts": {
+    "technical": {
+      "name": "",
+      "email": "",
+      "phone": "",
+      "person": null,
+      "extra": null
     },
-    "testnet": {
-      "common_name": "",
-      "endpoint": "",
-      "dns_names": []
+    "administrative": {
+      "name": "",
+      "email": "",
+      "phone": "",
+      "person": null,
+      "extra": null
     },
-    "mainnet": {
-      "common_name": "",
-      "endpoint": "",
-      "dns_names": []
+    "legal": {
+      "name": "",
+      "email": "",
+      "phone": "",
+      "person": null,
+      "extra": null
     },
-    "state": {
-        "current": 1,
-        "ready_to_submit": false,
-        "started": "",
-        "steps": [
-            {
-                "key": 1,
-                "status": "progress"
-            }
-        ]
+    "billing": {
+      "name": "",
+      "email": "",
+      "phone": "",
+      "person": null,
+      "extra": null
     }
+  },
+  "trixo": {
+    "primary_national_jurisdiction": "",
+    "primary_regulator": "",
+    "financial_transfers_permitted": "no",
+    "other_jurisdictions": [],
+    "has_required_regulatory_program": "no",
+    "conducts_customer_kyc": false,
+    "kyc_threshold": 10,
+    "kyc_threshold_currency": "USD",
+    "must_comply_travel_rule": false,
+    "applicable_regulations": [
+      "FATF Recommendation 16"
+    ],
+    "compliance_threshold": 3000,
+    "compliance_threshold_currency": "USD",
+    "must_safeguard_pii": false,
+    "safeguards_pii": false
+  },
+  "testnet": {
+    "common_name": "",
+    "endpoint": "",
+    "dns_names": []
+  },
+  "mainnet": {
+    "common_name": "",
+    "endpoint": "",
+    "dns_names": []
+  },
+  "state": {
+    "current": 1,
+    "ready_to_submit": false,
+    "started": "",
+    "steps": [
+      {
+        "key": 1,
+        "status": "progress"
+      }
+    ]
+  }
```

Comment:
Auto formatting from my IDE, please confirm that the only change is changing the `business_category` value.